package.json

```diff
@@ -18,8 +18,9 @@
     "react-dom": "^16.8.0-0"
   },
   "dependencies": {
+    "prop-types": "^15.7.2",
     "memoize-one": "^5.1.1",
-    "prop-types": "^15.7.2"
+    "regenerator-runtime": "^0.13.3"
```

Comment:
should this be a devDependency?

Reply:
Good catch. These tests are all about to be updated shortly to use https://github.com/testing-library/react-hooks-testing-library instead of @testing-library/react

Reply:
Updated 👍